### ISSUE TITLE
App doesn't require access to all the permission it supports in the trust policy file

### ISSUE THREAD

**Issue by irizzant** [octo-sts/app#581]
The list of available permissions to ask in a trust policy file is wider than the one requested by the installed app.

See for example security_events, which is not accessible by the app https://github.com/octo-sts/app/blob/bae5250f1daf30e647a13ab75c20863746ba3e6f/iac/sts_exchange.schema.json#L274

**Comment by mattmoor** [comment]
Yes, because this field's type literally comes from Google's library for the GitHub API, so the permissions we would need to ask for is "literally everything".

We plan to do quarterly releases and roll up new permissions into those releases (since it requires folks to hit "re-install" on the app), so if you have requests for permissions you need, please let us know.

@cpanato did we document ☝️ somewhere?

**Comment by cpanato** [comment]
no, i will do that
